static/js/bundles/app.js

```diff
@@ -199,6 +199,7 @@ import "../settings_ui.js";
 import "../search_pill.js";
 import "../search_pill_widget.js";
 import "../stream_ui_updates.js";
+import "../zpoilers.js";
```

Comment:
Cute name, but we should probably just call it `spoilers.js` for better readability :)

Reply:
😢  Aww, OK. We can definitely change it back (and it shouldn't be anywhere user-facing)